app/views/merchants/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @merchant.name %></h1>
+
+<% @merchant_products.each do |product| %>
+  <div class="col-md-4">
+    <ul style="list-style: none;">
```

Comment:
This style should be moved out of the markup and into a CSS class or something along those lines